ZenPacks/zenoss/CalculatedPerformance/ReadThroughCache.py

```diff
@@ -280,7 +280,14 @@ def batchFetchMetrics(self, datasources):
                 else:
                     dsPoints.add(dsdpID)
         metrics = {}
+        max_cycletime = 0
         for datasource in datasources:
+            try:
+                if datasource.cycletime > max_cycletime:
+                    max_cycletime = datasource.cycletime
+            except Exception:
```

Comment:
From what I can see the block might cause only AttributeError, Exception is too broad.

Reply:
Good point. Fixed.